### ISSUE TITLE
Merge Linux kernel 5.4.228

### ISSUE THREAD

**Issue by FuccDucc** [moto-sm7325/android_kernel_motorola_sm7325#1]
[Linux kernel 5.4.238](https://lwn.net/Articles/926875/), or the equivalent of latest security patch for Linux 5.4 at the time of this issue being handled, must be merged

Problem is that currently the Linux kernel version 5.4.191 is in this repository (as per 2908e85), and there's been important security updates since

**Comment by FuccDucc** [comment]
here @ThEMarD

**Comment by ThEMarD** [comment]
Heyyo @FuccDucc , tbh in Qualcomm kernels? Linux kernel merges aren't super crazy important as Qualcomm don't use a lot of the Linux kernel stuff as they provide their own solutions for most of it, so there's very little that might actually be implemented for us. Some devs even prefer to just keep a kernel at CLO Tags only which I believe is the direction we are going in as the last Linux kernel merge caused a few bugs we had to review and fix for us...

**Comment by FuccDucc** [comment]
> as the last Linux kernel merge caused a few bugs we had to review and fix for us...
@ThEMarD 

yes, but if you keep it maintained (instead of a huge backlog much later) it can be more doable. Better small bits than a lot of changes, all at once. I mean, the mentioned last update was less than a year ago, and very large as it was a big catch up. I will see if i find the time to do it myself with a PR instead of only asking, if the others agree there is a will to investigate and fix potential bugs that arise from it, of course i will be available for that as well.

Now, it would be version 5.4.238, and the diff between 5.4.191 and 5.4.238 isn't by far as large as 2908e85 was, so there is even less of a bug fixing operation as you experienced last time i'd imagine

**Comment by ThEMarD** [comment]
Heyyo, tbh I'm a kernel noob so I don't think I'd be successful doing so... LineageOS team already have a good kernel base we can use with latest Linux upstream and CodeLinaro tag merged too is what I'd recommend for a base and redo our changes on top, so in the future we can just merge any new changes on this repo

https://github.com/LineageOS/android_kernel_qcom_sm8350

I know our repo is called sm7335, but our soc yupik is based on Lahaina which is sm8350 and our kernel already includes motorola source code for sm8350 devices too (tundra, xpeng)
